mubench.reviewsite/templates/review.phtml

```diff
@@ -335,6 +397,18 @@ use MuBench\ReviewSite\Model\Experiment;
                     }
                     b.type = "hidden";
                 }
+                function onInput() {
+                    var tag_name_input = document.getElementById("tag-input").value;
+                    var suggested_tags = document.getElementById("tags").childNodes;
+                    var form = document.getElementById("tag-form");
+                    // loops over tags in the autocompletion list to submit the form when one is clicked
```

Comment:
Maybe add this link: https://stackoverflow.com/questions/30022728/perform-action-when-clicking-html5-datalist-option as a reference for why we do it this way. Maybe there eventually a better way, which will hopefully be documented in the thread ;)